src/Scene.cc

```diff
@@ -180,6 +217,18 @@ void Scene::SetOriginVisual(const bool _enabled)
   this->dataPtr->originVisual = _enabled;
 }
 
+/////////////////////////////////////////////////
+void Scene::SetSky(const sdf::Sky &_sky)
+{
+  this->dataPtr->sky.reset(new sdf::Sky(_sky));
```

Comment:
Can we use `make_unique` on all the places where there's currently `reset`?

Reply:
done. 579b521